domains/jonathan.json

```
@@ -0,0 +1,11 @@
+{
+  "description": "Portfolio website.",
+  "repo": "https://github.com/jonasfroeller/jonasfroeller",
+  "owner": {
+    "username": "JonathanB500",
+    "email": "jonathanbuitrago505@gmail.com"
+  },
+  "record": {
+    "CNAME": "https://jonathanbuitragoroncancio.web.app/"
```

CNAME cannot contain https or /
```suggestion
    "CNAME": "jonathanbuitragoroncancio.web.app"
```